app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx

```diff
@@ -15,8 +15,9 @@
  */
 
 import IconSVG from 'components/shared/IconSVG';
-import React from 'react';
```

Comment:
Other than adding the `PipelineConfigure` component here, are there any changes to this file?

Reply:
added more props from angular ctrl